main/streams/userspace.c

```diff
@@ -1382,6 +1382,8 @@ static int php_userstreamop_cast(php_stream *stream, int castas, void **retptr)
 	php_stream * intstream = NULL;
 	int call_result;
 	int ret = FAILURE;
+	/* If we a checking if the stream can cast, no return pointer is provided, so do not emit errors */
+	bool report_errors = retptr;
```

Comment:
Seems a bit hacky but I guess it's still better that hold call state in the stream struct so I'm cool with this. Not sure if we need temp var but not a big issue with that either...